packages/client/tests/functional/extensions/__snapshots__/itx.ts.snap

```diff
@@ -1,185 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`extensions.itx (provider=cockroachdb) extended client in itx can rollback via custom call 1`] = `
```

Comment:
This is a lot of snapshots being removed with no changes in the test suite, and they all seem to be related to the query compiler and driver adapters, and not the other way around. What's the story here?